### DISCUSSION TITLE
How to filter messages of a specific account from a stream? in new version tweepy

### DISCUSSION THREAD

**Discussion by CryDev1l** [tweepy/tweepy#1917] [Questions] [answered]
Can you please tell me how the stream.filter(follow=user_ids) construction has changed after updating tweepy versions?
This one has been looking for an answer to this question for about a week, no one can give an answer to it. Please, help!

**Comment by CryDev1l** [comment]
I still can't figure out how to use tweepy to filter the messages of the users I need in stream mode. Does anyone have a solution or thoughts on this?

**Comment by Harmon758** [comment, marked as answer]
[`Stream.filter`](https://tweepy.readthedocs.io/en/v4.11.0/stream.html#tweepy.Stream.filter) still accepts a `follow` parameter in all the latest versions of Tweepy (up to v4.11 right now).
Nothing has really changed in that regard.
